core/README.md

```diff
@@ -84,10 +84,7 @@ towards pushing Stellar's protocol development forward adhere to the following:
   aligns with those goals and values, it's unlikely to ever be implemented.
 - Gather feedback from discussion on the dev mailing list and other forums, and utilize it to begin
   a draft proposal, otherwise known as a CAP (Core Advancement Proposal).
-- Follow the proposal process listed below. If you're having difficulty moving the proposal
-  forward, talk to the buddy that's assigned the CAP; they'll often have guidance on how to move
-  things forward, as well as feedback regarding feasibility and how the proposal does or does not
-  align with the Stellar protocol's goals and values.
+- Follow the proposal process listed below
```

Comment:
This sentence looks incomplete, maybe by accident, I'm not sure, or at least is missing a full stop.
```suggestion
- Follow the proposal process listed below.
```